cypress/integration/tx_modal.spec.js

```diff
@@ -0,0 +1,203 @@
+const TEST_SAFE = 'rin:0x11Df0fa87b30080d59eba632570f620e37f2a8f7'
+const RECIPIENT_ENS = 'diogo.eth'
+const RECIPIENT_ADDRESS = '0x6a5602335a878ADDCa4BF63a050E34946B56B5bC'
+const SAFE_NONCE = '6'
+
+describe('Tx Modal', () => {
+  before(() => {
+    cy.connectE2EWallet()
+
+    // Open the Safe used for testing
+    cy.visit(`/${TEST_SAFE}`)
+    cy.contains('a', 'Accept selection').click()
+  })
+
+  describe('Send funds modal', () => {
+    describe('Send Funds form', () => {
+      before(() => {
+        // Open Send Funds Modal
+        cy.contains('New Transaction').click()
+        cy.contains('Send funds').click()
+      })
+
+      it('should display Send Funds modal with all the form elements', () => {
+        // Modal header
+        cy.get('form')
+          .prev()
+          .prev()
+          .within(() => {
+            // Send Funds modal is open
+            cy.contains('Send funds').should('be.visible')
+            cy.contains('Step 1 of 2').should('be.visible')
+
+            // Current network is same as Safe
+            cy.contains('Rinkeby').should('be.visible')
+          })
+
+        // It contains the form elements
+        cy.get('form').within(() => {
+          // Sending from the current Safe address
+          const [chainPrefix, safeAddress] = TEST_SAFE.split(':')
+          cy.contains(chainPrefix)
+          cy.contains(safeAddress)
+
+          // Current ETH balance
+          cy.contains('Balance:').contains('0.3 ETH')
```

Comment:
Still not sure if we should test for exact values or if we should just make sure there is something + ETH

Reply:
This safe is not expected to move funds so I think that's fine.